components/deleteButton.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { Button, Container, Form, InputField, Row, Spacer } from "@yakad/ui";
+import { useRouter } from "next/navigation";
+import { useFetch, useFormDataHandle } from "@yakad/lib";
+import React from "react";
+
+interface Delete {
+    controller: string;
+    uuid: string;
+}
+
+export default function DeleteButton() {
+    const router = useRouter();
+    const [formData, setFormData] = React.useState<Delete>();
+
+    const fetch = useFetch(
+        `${process.env.NEXT_PUBLIC_API_URL}/?{controller/?{uuid}`,
+        {
+            method: "DELETE",
+            headers: {
+                Accept: "application/json",
+                "Content-Type": "application/json",
```

Comment:
We dont send json in delete method, so i think we can remove entire header, @AzerothA please answer this.

Reply:
Yes, we should remove the entire header.